install/0000_30_machine-api-operator_12_baremetalhost.crd.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: baremetalhosts.metal3.io
```

Comment:
We need to figure out if we want to ship in this namespace.  @enxebre @derekwaynecarr @eparis

Reply:
the resource is alpha, i would like to use the community project name for these resources in the interim.  before graduating to GA in openshift, i want to understand how these API resources are evolved in community context.